sw/tools/install.py

```diff
@@ -0,0 +1,178 @@
+
+import lsb_release
+import subprocess
+import webbrowser
+
+release = lsb_release.get_distro_information()
+docs = 'https://paparazzi-uav.readthedocs.io'
+
+
+class InstallWindow(QWidget):
+
+    def execute(self,cmd):
+        self.buttonlist.setEnabled(False)
+        print(cmd)
+        p = subprocess.Popen(cmd, shell=True)
+        QApplication.processEvents()
+        p.communicate() # wait until ready
+        print('================================================================================')
+        print(cmd)
+        print('READY (verify above)')
+        print('Check documentation to resolve problems:', docs)
+        print('================================================================================')
+        self.buttonlist.setEnabled(True)
+        
+
+    def view(self,link):
+        webbrowser.open(link)
+
+    def cmd_repo(self):
+        self.execute('sudo add-apt-repository -y ppa:paparazzi-uav/ppa && sudo apt-get update')
+
+
+    def cmd_dev(self):
+        self.execute('sudo apt-get -f -y install paparazzi-dev')
+        # Missing
+        if float(release['RELEASE']) <= 20.04:
+            self.execute('sudo apt-get install python3-lxml python3-numpy')
+
+    def cmd_arm(self):
+        self.execute('sudo apt-get -f -y install paparazzi-dev')
+        if float(release['RELEASE']) >= 20.04:
+            self.execute('sudo apt-get install python-is-python3 gcc-arm-none-eabi gdb-multiarch')
+            # python3-serial?
+        else:
+            self.execute('sudo add-apt-repository -y ppa:team-gcc-arm-embedded/ppa')
+            self.execute('sudo apt-get update')
+            self.execute('sudo apt-get -f -y install gcc-arm-embedded')
+
+    def cmd_jsb(self):
+        self.execute('sudo apt-get -f -y install paparazzi-jsbsim')
+
+    def cmd_gcs(self):
+        self.execute('sudo apt-get -f -y install pprzgcs')
+
+    def cmd_mcu(self):
+        self.execute('sudo apt-get -f -y install dfu-util')
+        self.execute('sudo cp conf/system/udev/rules/*.rules /etc/udev/rules.d/ && sudo udevadm control --reload-rules')
+
+    def cmd_gazebo(self):
+        self.execute('sudo apt-get -f -y install gazebo9 libgazebo9-dev')
+        self.execute('git submodule init && git submodule sync && git submodule update ./sw/ext/tudelft_gazebo_models')
+
+
+    def cmd_bebopcv(self):
+        self.execute('git submodule init && git submodule sync && git submodule update ./sw/ext/opencv_bebop')
+        self.execute('sudo apt-get -f -y install cmake libjpeg-turbo8-dev libpng-dev libtiff-dev zlib1g-dev libdc1394-22-dev')
+
+    def cmd_vlc(self):
+        self.execute('sudo apt-get -f -y install ffmpeg vlc jstest-gtk default-jre')
+        self.execute('sudo apt-get install python3-pip')
+        if float(release['RELEASE']) <= 20.04:
+            self.execute('pip3 install pyquaternion ivy-python') # Required for NatNat
+        else:
+            self.execute('pip3 install pyquaternion ivy') # Required for NatNat
```

Comment:
```suggestion
        self.execute('pip3 install pyquaternion ivy-python') # Required for NatNat
```
`ivy` is something completly different: https://pypi.org/project/ivy/

Reply:
Ok, indeed, ivy was not needed in 22.04 and `ivy-python` is part of `paparazzi-dev`. I did need `pyquaternion` and `ivy-python` in 18.04 for sure. On the others, I should re-check (reinstall fresh ubuntu). If the package was already installed nothing happens so for now we can leave it.